src/sonic-yang-models/yang-events/sonic-events-bgp.yang

```diff
@@ -0,0 +1,97 @@
+module sonic-events-bgp {
+    namespace "http://github.com/Azure/sonic-events-bgp";
+
+    yang-version 1.1;
+
+    import sonic-events-common {
+        prefix evtcmn;
+    }
+
+    import openconfig-alarm-types {
+        prefix oc-alarm-types;
+    }
+
+    revision 2022-12-01 {
+        description "BGP alert events.";
+    }
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description
+        "SONIC BGP events";
+
+    container bgp-state {
+        oc-alarm-types:MINOR
+
+        description "
+            Declares an event for BGP state for a neighbor IP going up/down.;
+
+        leaf ip {
+            type inet:ip-address;
+            description "IP of neighbor";
+        }
+
+        leaf status {
```

Comment:
We can also check, if we have common types for this in sonic_types.yang

Reply:
YANG review is moved to different [PR #11404 ](https://github.com/Azure/sonic-buildimage/pull/11404)